packages/@aws-cdk/aws-iot/README.md

```diff
@@ -83,3 +99,5 @@ new iot.TopicRule(this, 'TopicRule', {
   enabled: false,
 });
 ```
+
+See also [@aws-cdk/aws-iot-actions](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-iot-actions/README.md) for other actions.
```

Comment:
Let's use a link to our API reference (https://docs.aws.amazon.com/cdk/api/latest/docs/aws-iot-actions-readme.html) instead of GitHub.